.github/ISSUE_TEMPLATE/libmamba-bug-report.yml

```diff
@@ -0,0 +1,84 @@
+name: Libmamba Bug Report (Experimental Feature)
+description: Create a report for the experimental solver
+labels: ['solver::libmamba', 'tag::experimental']
```

Comment:
We'll need to update the https://github.com/conda/conda/blob/master/.github/labels.yml file with the "tag::experimental" label.

Reply:
Do we really need a new tag for this? Are we trying to capture a difference between something being https://github.com/conda/conda/labels/solver%3A%3Alibmamba vs. https://github.com/conda/conda/labels/solver%3A%3Alibmamba + `[tag::experimental]`?

Reply:
You're right, it's fine to just use [solver::lib](https://github.com/conda/conda/labels/solver%3A%3Alibmamba).

Reply:
We are going to have more experimental features and items in the future, if like to capture if they are generating more issues front the beginning rather than going back and retro actively applying

Reply:
Discussed with @LtDan33 and will define `[tag::experimental]` so we can group this and future experimental features together.